### PR TITLE
Restore v1 keymaps for tab switching

### PR DESCRIPTION
**Pull request by chabou**
This PR restore keymaps used in v1 for tab switching: https://github.com/zeit/hyper/blob/57166a57a2361894f8897630fff6d460e96aae3f/lib/containers/hyper.js#L57-L64

### CODE REVIEW

File: app/keymaps/linux.json [hunk 1]
```diff
@@ -36,4 +46,4 @@
   "editor:deleteEndLine": "ctrl+end",
   "editor:clearBuffer": "ctrl+shift+k",
   "plugins:update": "ctrl+shift+u"
-}
+}
```
Comment (albinekb) [inline app/keymaps/linux.json]: Missing newline

File: app/keymaps/win32.json [hunk 2]
```diff
@@ -36,4 +46,4 @@
   "editor:deleteEndLine": "ctrl+end",
   "editor:clearBuffer": "ctrl+shift+k",
   "plugins:update": "ctrl+shift+u"
-}
+}
```
Comment (albinekb) [inline app/keymaps/win32.json]: Missing newline

File: app/keymaps/darwin.json [hunk 3]
```diff
@@ -37,4 +47,4 @@
   "editor:clearBuffer": "command+k",
   "editor:emojis": "command+ctrl+space",
   "plugins:update": "command+shift+u"
-}
+}
```
Comment (albinekb) [inline app/keymaps/darwin.json]: Missing newline

**Comment by rauchg** [comment]
🙏

**Pull request merged**